applications-dev/plugins/SofaQtQuickGUI/SofaViewer.cpp

```diff
@@ -505,7 +505,7 @@ QPair<QVector3D, QVector3D> SofaViewer::boundingBox() const
 QPair<QVector3D, QVector3D> SofaViewer::rootsBoundingBox() const
 {
 	QVector3D min, max;
-	mySofaScene->computeBoundingBox(min, max, myRoots); /*attention, ne prend en compte que le dernier �l�ment mis dans la liste ?*/
+	mySofaScene->computeBoundingBox(min, max, myRoots); /*attention, ne prend en compte que le dernier �l�ment mis dans la liste ?*/
```

Comment:
I guess it should translated in english....

Reply:
Ho yes this is why my editor was complaining all the time.